hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmDBAccessIdInfo.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.hadoop.ozone.om.helpers;
+
+import com.google.common.base.Preconditions;
+import org.apache.hadoop.hdds.StringUtils;
+
+/**
+ * This class is used for storing Ozone tenant accessId info.
+ */
+public final class OmDBAccessIdInfo {
+  /**
+   * Name of the tenant.
+   */
+  private final String tenantId;
+  /**
+   * Kerberos principal this accessId belongs to.
+   */
+  private final String kerberosPrincipal;
+  /**
+   * Shared secret of the accessId. TODO: Encryption?
+   */
+  private final String sharedSecret;
+
+  // This implies above String fields should NOT contain the split key.
+  public static final String SERIALIZATION_SPLIT_KEY = ";";
```

Comment:
this is fine since POSIX systems are supposed to support user names composed of " lower and upper case ASCII letters, digits, period, underscore, and hyphen". Windows systems do not support ";" as a valid user name character.